src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -39,7 +40,7 @@ class QuestionHelper extends Helper
      * @param OutputInterface $output   An OutputInterface instance
      * @param Question        $question The question to ask
      *
-     * @return string The user answer
+     * @return string|array|mixed The user answer
```

Comment:
`|mixed` looks strange since it logically includes `string|array`. Is there a way to say "in this case, this is returned, etc." to hint the user into something useful?